src/bin/pgcopydb/ld_wal2json.c

```diff
@@ -95,6 +95,40 @@ parseWal2jsonMessageActionAndXid(LogicalStreamContext *context)
 		return false;
 	}
 
+	/*
+	 * if metadata action is one of INSERT, UPDATE, DELETE, TRUNCATE
+	 * then we need to parse the schema and table name from the message
+	 * and check if the message should be filtered out
+	 */
```

Comment:
I thought we could use the wal2json filtering abilities and append the list of tables we filter-out to the proper option when creating the replication slot with the wal2json plugin. Did you try that approach? Is there a learning where we can't use that option and have to filter-out in our code here? Do you think maybe we need to do both (defensive programming)?